app/src/main/java/com/project/lab1/feed/FeedActivity.kt

```diff
@@ -0,0 +1,42 @@
+package com.project.lab1.feed
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.project.lab1.R
+import com.project.lab1.feed.adapters.FeedRecyclerViewAdapter
+import com.project.lab1.feed.models.FeedItem
+import kotlinx.android.synthetic.main.activity_main.*
+
+class FeedActivity : AppCompatActivity() {
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var viewAdapter: RecyclerView.Adapter<*>
+    private lateinit var viewManager: RecyclerView.LayoutManager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_feed)
```

Comment:
Change init of activity view to android view binding https://developer.android.com/topic/libraries/view-binding